pkg/networkmanager/interfaces.js

```diff
@@ -488,15 +499,20 @@ export function NetworkManagerModel() {
                 return def;
         }
 
-        function get_ip(first, addr_from_nm, route_from_nm, ip_to_text) {
+        function get_ip(first, ip_to_text) {
+            const dns_data = self.supports_dns_data
+                ? get(first, "dns-data", [])
+                : get(first, "dns", []).map(ip_to_text);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22330-4f4b27ef-20250909-073142-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)